R/demo/bonds.R

```diff
@@ -237,11 +236,25 @@ invisible(setCouponPricer(Bond_cashflows(floatingRateBond), pricer))
 
 
 ## Yield curve bootstrapping
-invisible(RelinkableQuoteHandle_linkTo(forecastingTermStructure, depoSwapTermStructure))
-invisible(RelinkableQuoteHandle_linkTo(discountingTermStructure, bondDiscountingTermStructure))
+if (FALSE) {
+  # equivalent to the method in else
+  invisible(forecastingTermStructure$linkTo(depoSwapTermStructure))
+  invisible(discountingTermStructure$linkTo(bondDiscountingTermStructure))
+} else {
+  # equivalent to the previous method
+  invisible(RelinkableYieldTermStructureHandle_linkTo(forecastingTermStructure, depoSwapTermStructure))
+  invisible(RelinkableYieldTermStructureHandle_linkTo(discountingTermStructure, bondDiscountingTermStructure))
+}
```

Comment:
Which one is more idiomatic?

Reply:
Hallo,
swig is generating S4 clases, so the idiomatic form would be `methodName(self, args)`.
However swig defines also a `$` operator which allows users to invoke methods as if they were R5 classes.

Thanks

Reply:
Ok, thanks.  I'll just keep the `methodName(self, args)` then, since the rest of the file seems to be in this style.  There are other examples (like european-option.R) which demonstrate the other style.